File: CPP/Product.cpp
class Product
{
	// private attributes
	private:
		int price;
		string idProduct;

	// public attributes
	public:
		// constructor with parameter
		Product(int price, string idProduct)
		{
			this->price = price;
			this->idProduct = idProduct;
		}
		// constructor
		Product()
		{
			this->price = 0;
			this->idProduct = "";
		}

		// method for price attribute
		void setPrice(int price)
		{
			this->price = price;
		}
		int getPrice(){
			return this->price;
		}

		// method for idProduct attribute
		void setIDProduct(string idProduct)
		{
			this->idProduct = idProduct;
		}
		string getIDProduct()
		{
			return this->idProduct;
		}

		// destructor
		~Product()
		{

		}
};

File: PHP/Product.php
<?php 

    // class for Product
    class Product
    {
        // public attributes
        public $idProduct;
        public $price;

        // constructor
        public function __construct($idProduct = "", $price = 0)
        {
            $this->idProduct = $idProduct;
            $this->price = $price;            
        }

        // methods for id product attribute
        public function setIDProduct($idProduct)
        {
            $this->idProduct = $idProduct;
        }   
        public function getIDProduct()
        {
            return $this->idProduct;
        }

        // methods for price attribute
        public function setPrice($price)
        {
            $this->price = $price;
        }   
        public function getPrice()
        {
            return $this->price;
        }

        // destructor
        public function __destruct()
        {
            
        }
    }
?>